TradeItIosTicketSDK2/TradeItPortfolioAccountDetailsViewController.swift

```diff
@@ -0,0 +1,102 @@
+import UIKit
+import MBProgressHUD
+
+class TradeItPortfolioAccountDetailsViewController: TradeItViewController, TradeItPortfolioAccountDetailsTableDelegate {
+    var tableViewManager = TradeItPortfolioAccountDetailsTableViewManager()
+    var linkBrokerUIFlow = TradeItLinkBrokerUIFlow()
+    var tradingUIFlow = TradeItTradingUIFlow()
+    var alertManager = TradeItAlertManager()
+    var activityView: MBProgressHUD?
+    var linkedBrokerAccount: TradeItLinkedBrokerAccount?
+
+    @IBOutlet weak var table: UITableView!
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        guard let linkedBrokerAccount = self.linkedBrokerAccount else {
+            preconditionFailure("TradeItIosTicketSDK ERROR: TradeItPortfolioViewController loaded without setting linkedBrokerAccount.")
+        }
+
+        self.navigationItem.title = linkedBrokerAccount.linkedBroker?.brokerName
+
+        self.tableViewManager.delegate = self
+        self.tableViewManager.table = self.table
+
+        self.activityView = MBProgressHUD.showAdded(to: self.view, animated: true)
+        self.activityView?.label.text = "Authenticating"
+
+        linkedBrokerAccount.linkedBroker?.authenticateIfNeeded(onSuccess: {
+            self.activityView?.label.text = "Fetching data"
+
+            linkedBrokerAccount.getAccountOverview(onSuccess: { _ in
+                self.activityView?.hide(animated: true)
+                self.tableViewManager.updateAccount(withAccount: linkedBrokerAccount)
+            }, onFailure: { errorResult in
+                self.activityView?.hide(animated: true)
+                // TODO: Figure out error handling
+                print(errorResult)
+            })
+
+            linkedBrokerAccount.getPositions(
+                onSuccess: { positions in
+                    self.activityView?.hide(animated: true)
+                    self.tableViewManager.updatePositions(withPositions: positions)
+                }, onFailure: { errorResult in
+                    self.activityView?.hide(animated: true)
+                    // TODO: Figure out error handling
+                    print(errorResult)
+                }
+            )
```

Comment:
I didn't make this promise based because they are both distinct requests - although it would be nice to have a single error handler.